spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -16,15 +16,16 @@
 
 package org.springframework.integration.zip.splitter;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
 import org.apache.commons.io.FilenameUtils;
 import org.springframework.integration.file.FileHeaders;
 import org.springframework.integration.support.MessageBuilder;
 import org.springframework.integration.zip.ZipHeaders;
 import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
We use a bit different `import` order:
![imports](https://cloud.githubusercontent.com/assets/1071590/14219422/393c7440-f828-11e5-802b-e75139f9a5a9.jpg)

You can simply reconfigure your IDE to follow with our rules.
Or just let us know that we will forced to revert such a change on merge each time :wink: